Database.cpp

```diff
@@ -1,8 +1,44 @@
 #include "Database.hpp"
+#include <algorithm>
```

Comment:
It seems to me that <algorithm> should be inside Database.hpp. I can be wrong ;)